tdrs-frontend/src/components/SiteMap/SiteMap.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useRef, useState } from 'react'
+
+const SiteMap = () => (
+  <div className="margin-top-5">
+    <SiteMap.Link text="Home" link="/home" />
```

Comment:
Can you explain what is going on here? You are referencing the SiteMap variable as it's building itself?